03_build_installer.sh

```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env bash
+
 set -x
 set -e
 
 source logging.sh
 source utils.sh
 source common.sh
+if [ "$NODES_PLATFORM" = "assisted" ]; then
```

Comment:
I think you should create a new make target which just doesn't run these scripts

Reply:
@hardys but then it will different flow than usual dev scripts. I didn't want to have it as different flow. I want to match same readme

Reply:
The docs need updating anyway to describe the assisted workflow. Since you need to skip entire steps, I think it's appropriate to have a different target.

Reply:
@stbenjam you are right regarding docs but not sure i agree with different target(Not bad to have it though). On my opinion the idea to have config file is to be able to use same targets for different types of platforms.
Main idea was to give user the chance to have the same experience as he had before but with new platform.